var/spack/repos/builtin/packages/fontconfig/package.py

```diff
@@ -35,6 +35,7 @@ class Fontconfig(AutotoolsPackage):
     version('2.11.1', 'e75e303b4f7756c2b16203a57ac87eba')
 
     depends_on('freetype')
+    depends_on('gperf', type='build')
```

Comment:
@adamjstewart -- See [the fontconfig bug in which I ask this question](https://bugs.freedesktop.org/show_bug.cgi?id=101546).

I'll follow up.